scripts/mbedtls_dev/bignum_core.py

```diff
@@ -72,29 +72,54 @@ def generate_function_tests(cls) -> Iterator[test_case.TestCase]:
             yield cls(a_value, b_value).create_test_case()
 
 
+class BignumCoreOperationArchSplit(BignumCoreOperation):
+    #pylint: disable=abstract-method
+    """Common features for bignum core operations where the result depends on
+    the limb size."""
+
+    def __init__(self, val_a: str, val_b: str, bits_in_limb: int) -> None:
+        super().__init__(val_a, val_b)
+        bound_val = max(self.int_a, self.int_b)
+        self.bits_in_limb = bits_in_limb
+        self.bound = bignum_common.bound_mpi(bound_val, self.bits_in_limb)
+        limbs =  bignum_common.limbs_mpi(bound_val, self.bits_in_limb)
+        byte_len = limbs*self.bits_in_limb//8
```

Comment:
(minor, whitespace) this line and next: binary operators should presumably have a space either side